<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lecteurvue</class>
 <widget class="QMainWindow" name="lecteurvue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lecteur de diaporama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbImage">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bLancer">
        <property name="text">
         <string>Lancer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bArreter">
        <property name="text">
         <string>Arreter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bPrecedent">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSuivant">
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bManuel">
        <property name="text">
         <string>Manuel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bAuto">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_Param_tre">
    <property name="title">
     <string>&amp;Paramètre</string>
    </property>
    <addaction name="actionCharger_diaporama"/>
    <addaction name="actionEnlever_diaporama"/>
    <addaction name="actionVitesse_de_defilement"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Param_tre"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVitesse_de_defilement">
   <property name="text">
    <string>Vitesse de défilement</string>
   </property>
  </action>
  <action name="actionEnlever_diaporama">
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
  </action>
  <action name="actionCharger_diaporama">
   <property name="text">
    <string>Charger diaporama</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de...</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
